lib/rspec/core/option_parser.rb

```diff
@@ -112,7 +112,7 @@ def parser(options)
                   '  [d]ocumentation (group and example names)',
                   '  [h]tml',
                   '  [j]son',
-                  '  [m]inimal (suitable for editors integration)',
+                  '  [f]ailure list (suitable for editors integration)',
```

Comment:
The full name here (`failure list`, on this line) needs to match the option in the case statement in `formatters.rb` where you used `failure_list`.  Instead of having to decide space vs underscore vs dash or whatever, WDYT of just calling the option `failures`?

Reply:
That's a good point from @myronmarston, and `failure list` will probably trick users who do not understand how the shell process arguments and don't escape the space char.

I'll add that a hint to the user of the format of the output would help him figure-out what it means without needing to try it:

`[f]ailures ("file:line:reason", suitable for editors integration)`

Reply:
Sorry for the mistake.

Good point. Do you think we should then rename `FailureListFormatter` to  `FailuresListFormatter`?

Reply:
No. Grammatically thats looks funny. A list of failures is a failure list.

Reply:
And don't apologise, I'm to blame for rushing the merge :joy:

Reply:
Here we go : https://github.com/rspec/rspec-core/pull/2625 

I also adde the good suggestion of @smortex. Thanks for your feedbacks